inst/create_cm_template.R

```diff
@@ -0,0 +1,230 @@
+# Name: CM domain
+#
+# Label: R program to create CM Domain
+#
+# Input raw data:
+# study_sdtm_spec
+# study_controlled_terminology
+# study_raw_datasets
+#
+#
+
+library(sdtm.oak)
+library(dplyr)
+
+# Read Specification
+
+# sdtm_spec <- read_sdtm_spec(filename = "~/study/study_sdtm_spec.csv")
+
+study_ct <- read_study_ct(filename = "~/study/study_sdtm_ct.csv")
+
+# Read in raw data
+
+MD1 <- read_raw_data_csv(filename = "~/study/MD1.csv") |>
```

Comment:
Is `MD1.csv` the long lean format we agreed on? Or we changed the input format at some point?

Reply:
We can skip the long, lean format for now. Based on my research, we can develop the algorithms as they are.